cime_config/cesm/machines/config_machines.xml

```diff
@@ -218,18 +218,18 @@
     <mpirun mpilib="mvapich2">
       <executable>srun</executable>
       <arguments>
-        <arg name="mpi">--mpi=none</arg>
```

Comment:
It looks like the rest of the file is using 2-space indenting. Why change that here?